Feature/improve vscode integration

manthonyaiello opened the pull request:
Overall simplification of the usage of `configure-vscode` + the ability to build through VS Code's menu / keyboard shortcut using the makefile + anod.

To test:
* clone this repo
* run the install script
* run `anod build`
* run `anod devel-setup uxas`
* open VS Code and point it to /path/to/bootstrap/develop/OpenUxAS
* Terminal > Run build command or Ctrl + Shift + B

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/AdaCore/OpenUxAS-bootstrap?pullRequest=8) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/AdaCore/OpenUxAS-bootstrap?pullRequest=8) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/AdaCore/OpenUxAS-bootstrap?pullRequest=8) it.</sub>

manthonyaiello committed:
Improve integration with VS Code

Building with VS Code through the Terminal menu is enabled by adding
`tasks.json` and providing proper configuration. As discovered working
with NetBeans, running our Makefile is trivial - but the correct python
environment has to be selected to ensure that anod can be used by the
Makefile to properly configure the build environment. Just as with the
`c_cpp_properties.json`, hard-coding the path to the python environment
and committing that to the repo seems wrong, so we don't do it. Instead,
we generate an appropriate `tasks.json` that has the path to bootstrap
in it.

This is all automated from the point of view of the user by chaining
anod-devel-setup to anod-configure-vscode when uxas is the target of
configuration. So the user doesn't have to worry about the extra command
in this case and can just get the benefit. If the user wants to manually
check out OpenUxAS, they can use configure-vscode for that repo.

This has been lightly tested and seems to be doing the right thing. It's
not terribly robust and it will overwrite an existing configuration, in
its current form.

manthonyaiello committed:
Documentation for the improved integration

pat-rogers reviewed (approved):
Cool!

manthonyaiello commented:
This repo is now abandoned.
